packages/eslint-plugin/test/no-deprecated-type-references.test.ts

```diff
@@ -138,6 +140,38 @@ ruleTester.run("no-deprecated-type-references", noDeprecatedTypeReferencesRule,
                 const mySelectProps: SelectProps = { items: [] };
             `,
         },
+
+        // Ensure that multiple references/uses of the _same_ deprecated type are all fixed
```

Comment:
can we add a test case that mixes both type reference + interface extending?

Reply:
done